disable ADE Framework

Currently trying to cross compile master-branch, with g++-4.8-arm-linux-gnueabihf

But tool-chain doesn't support ADE,

> [24%] Building CXX object modules/CMakeFiles/ade.dir/__/3rdparty/ade/ade-0.1.1c/sources/ade/source/alloc.cpp.o
In file included from /home/atmark/opencv/platforms/linux/build/3rdparty/ade/ade-0.1.1c/sources/ade/include/ade/util/math.hpp:14:0,
                from /home/atmark/opencv/platforms/linux/build/3rdparty/ade/ade-0.1.1c/sources/ade/source/alloc.cpp:16:
/home/atmark/opencv/platforms/linux/build/3rdparty/ade/ade-0.1.1c/sources/ade/include/ade/util/assert.hpp:27:1: error: expected unqualified-id before ‘[’ token
/home/atmark/opencv/platforms/linux/build/3rdparty/ade/ade-0.1.1c/sources/ade/include/ade/util/assert.hpp:29:1: error: expected unqualified-id before ‘[’ token
/home/atmark/opencv/platforms/linux/build/3rdparty/ade/ade-0.1.1c/sources/ade/include/ade/util/assert.hpp:86:1: error: expected unqualified-id before ‘[’ token
make[2]: *** [modules/CMakeFiles/ade.dir/__/3rdparty/ade/ade-0.1.1c/sources/ade/source/alloc.cpp.o]

Editing ADE source to remove [[noreturn]] is far from enough.
I dug the cmakelist but couldn't find an ADE related flag, is there anyway to build opencv4 without ADE ?

--- a/modules/gapi/cmake/init.cmake
+++ b/modules/gapi/cmake/init.cmake
@@ -1,3 +1,9 @@
+OCV_OPTION(WITH_ADE "Enable ADE framework (required for Graph API module)" ON)
+
+if(NOT WITH_ADE)
+  return()
+endif()
+
 if (ade_DIR)
   # if ade_DIR is set, use ADE-supplied CMake script
   # to set up variables to the prebuilt ADE

